src/main/kotlin/no/nav/bidrag/arbeidsflyt/model/constants.kt

```diff
@@ -3,10 +3,11 @@ package no.nav.bidrag.arbeidsflyt.model
 // Detaljer (keys) i en JournalpostHendelse
 object Detalj {
     const val FAGOMRADE = "fagomrade"
-    const val JP_ADM_ENHETSNUMMER = "enhetsnummer"
+    const val ENHETSNUMMER = "enhetsnummer"
+    const val ENHETSNUMMER_NYTT = "nytt-enhetsnummer"
```

Comment:
synkroniserer disse verdiene med siste endringer i `bidrag-dokument-journalpost`

Reply:
Hadde det ikke vært en fordel å legge dette inn som strukturerte data?
`{
  "journalforende_enhet": {
    "ny": "4865",
    "gammel": "4812"
  }
}`

Reply:
d høres ut som en bedre leselig løsning... dette må dog synkroniseres med `bidrag-dokument-journalpost`... to be continued...